draft-ietf-anima-brski-prm.md

```diff
@@ -701,12 +717,13 @@ If it still acts as server, the defined BRSKI-PRM endpoints to trigger a Pledge
 # Exchanges and Artifacts {#exchanges_uc2}
 
 The interaction of the pledge with the Registrar-Agent may be accomplished using different transports (i.e., protocols and/or network technologies).
-This specification utilizes HTTP as default transport.
+This specification utilizes HTTP(S) as default transport.
 Other specifications may define alternative transports such as CoAP, Bluetooth Low Energy (BLE), or Near Field Communication (NFC).
 These transports may differ from and are independent of the ones used between the Registrar-Agent and the registrar.
 
-Transport independence is realized through data objects that are not bound to specific transport security and stay the same along the communication path from the pledge via the Registrar-Agent to the registrar.
-Therefore, authenticated self-contained artifacts (e.g., JWS-signed JSON structures or COSE-signed CBOR structures) are used for the data exchanges between the pledge and the registrar via the Registrar-Agent.
+Transport independence is realized through authenticated self-contained objects that are not bound to a specific transport security and stay the same along the communication path from the pledge via the Registrar-Agent to the registrar.
+This specification utilizes JWS-signed JSON structures as default format for artifacts.
+Other specifications may define alternative formats such as CMS-signed JSON structures or COSE-signed CBOR structures.
```

Comment:
The formulation reads better!
Just one proposal to connect 724 and 725.
Proposal to enhance the statement in 725: 
 "This specification utilizes JWS-signed JSON structures as default format for artifacts representing authenticated self-contained objects."

Reply:
I like the cleared connection between the different terms!

I was not sure if we should mention CMS-signed JSON here, but felt I should mention more than COSE...

Reply:
Hm, Ijust thought it may be better to make the connection between JWS-signed JSON and authenticated self-contained objects explicit.

Regarding the other form factors, it is good to reference COSE. CMC-signed JSON is already used in RFC 8366. So maybe the sentence in 726 could be changed to 
"Other specifications may define alternative formats COSE-signed CBOR structures. {{I-D.ietf-anima-rfc8366bis}} already defines another format with CMS-signed JSON."

Reply:
Done